systemverilog-plugin/UhdmAst.cc

```diff
@@ -4824,6 +4951,7 @@ AST::AstNode *UhdmAst::process_object(vpiHandle obj_handle)
         // so the plugin doesn't need to process the parameter itself.
         // Other parameter types are handled by the plugin
         // mainly because they were implemented before Surelog did the substitution.
+        process_type_parameter();
```

Comment:
Is above comment still relevant? I think now we are processing type parameters, so it is no longer valid.

Reply:
I agree this is no longer valid. I've removed the comment.